index.html

```diff
@@ -0,0 +1,102 @@
+      <a href="#">Work</a>
+      <a href="#">Contact</a>
+    </div>
+  </nav>
+
+  <section id="welcome-section">
+    <h1>Hey I am Mimic</h1>
+    <p>a web developer</p>
+  </section>
+
+  <!-- project entry -->
+  <section id="projects">
+    <h2>This are some of my projects</h2>
+    <div class="project-container">
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" id="profile-link"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project01"></a>
+        <p>Tribute Page</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" id="profile-link"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project01"></a>
+        <p>Tribute Page</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+    </div>
+  </section>
+
+
+
+  <footer>
+    <div class="social-links">
+      <a href="#"><i class="fa fa-facebook-square"></i> Facebook</a>
+      <a href="#"><i class="fa fa-github"></i> Github</a>
+      <a href="#"><i class="fa fa-twitter"></i> Twitter</a>
+      <a href="#"><i class="fa fa-at"></i> Email</a>
+      <a href="#"><i class="fa fa-mobile"></i> Phone</a>
+    </div>
+
+    <div class="bottom-footer">
+      
+    </div>
+  </footer>
+
+
+
+
+
+
+
+
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
I noticed some unnecessary comments or code in your submission which can negatively affect the cleanliness and readability of the code. To improve its quality, kindly remove any unneeded comments or code.

Thank you for your effort on this project. Looking forward to seeing your updated code.